setup.py

```diff
@@ -1,34 +0,0 @@
-try:
-    from setuptools import setup
-except ImportError:
-    from distutils.core import setup
-
-readme = open('README.md').read()
-
-setup(
-    long_description=readme,
-    long_description_content_type='text/markdown',
-    name='blobby',
-    version='0.0.0',
-    description='This package provides a simple abstraction to an object store.',
```

Comment:
Massive nitpick (sorry): 'This package provides' seems somewhat superfluous - it's pretty clear that this a package and that it provides something. I think that `Simple object store abstraction` is clearer and easier to read :smile:

Reply:
Changing to `A library to provide a simple abstraction of an object store.` - the fact that it's a library not being inherent, unlike that it is a package :)